MAINTAINERS.md

```diff
@@ -0,0 +1,484 @@
+	   F:	net/
+	   X:	net/ipv6/
+	   matches all files in and below net excluding net/ipv6/
+	K: Keyword perl extended regex pattern to match content in a
+	   patch or file.  For instance:
+	   K: of_get_profile
+	      matches patches or files that contain "of_get_profile"
+	   K: \b(printk|pr_(info|err))\b
+	      matches patches or files that contain one or more of the words
+	      printk, pr_info or pr_err
+	   One regex pattern per line.  Multiple K: lines acceptable.
+        P: Peer for a component
+
+
+## General Project Architects
+
+    M: Amar Tumballi <amar@kadalu.io>
+    M: Xavier Hernandez  <xhernandez@redhat.com>
+    M: Pranith Karampuri <pranith.karampuri@phonepe.com>
+    P: Atin Mukherjee <atin.mukherjee83@gmail.com>
+
+## xlators:
+
+### Access Control List (ACL)
+    S: Maintained
+    F: xlators/system/posix-acl/
+
+### Arbiter
+    M: Ravishankar N <ravishankar@redhat.com>
+    P: Pranith Karampuri <pranith.karampuri@phonepe.com>
+    S: Maintained
+    F: xlators/features/arbiter/
+
+### Automatic File Replication (AFR)
+    M: Pranith Karampuri <pranith.karampuri@phonepe.com>
+    M: Ravishankar N <ravishankar@redhat.com>
+    P: Karthik US <ksubrahm@redhat.com>
+    S: Maintained
+    F: xlators/cluster/afr/
+
+### Barrier
+    S: Maintained
+    F: xlators/features/barrier
+
+### BitRot
+    M: Kotresh HR <khiremat@redhat.com>
+    P: Raghavendra Bhat <rabhat@redhat.com>
+    S: Maintained
+    F: xlators/features/bit-rot/
+
+### Changelog
+    M: Aravinda Vishwanathapura <aravinda@kadalu.io>
+    P: Kotresh HR <khiremat@redhat.com>
+    S: Maintained
+    F: xlators/features/changelog/
+
+### Distributed Hashing Table (DHT)
+    M: Mohit Agrawal <moagrawa@redhat.com>
+    M: Barak Sason Rofman <bsasonro@redhat.com>
+    M: Tamar Shacked <tshacked@redhat.com>
+    P: Csaba Henk <chenk@redhat.com>
+    S: Maintained
+    F: xlators/cluster/dht/
+
+### Erasure Coding
+    M: Pranith Karampuri <pranith.karampuri@phonepe.com>
+    M: Xavier Hernandez  <xhernandez@redhat.com>
+    P: Ashish Pandey <aspandey@redhat.com>
+    P: Sheetal Pamecha <spamecha@redhat.com>
+    S: Maintained
+    F: xlators/cluster/ec/
+
+### Error-gen
+    S: Maintained
+    F: xlators/debug/error-gen/
+
+### FUSE Bridge
+    M: Csaba Henk <chenk@redhat.com>
+    P: Niels de Vos <ndevos@redhat.com>
+    S: Maintained
+    F: xlators/mount/
+
+### Index
+    M: Pranith Karampuri <pranith.karampuri@phonepe.com>
+    P: Ravishankar N <ravishankar@redhat.com>
+    S: Maintained
+    F: xlators/features/index/
+
+### IO Cache
+    P: Mohammed Rafi KC <rafi.kavungal@iternity.com>
+    S: Maintained
+    F: xlators/performance/io-cache/
+
+### IO Statistics
+    S: Maintained
+    F: xlators/debug/io-stats/
+
+### IO threads
+    M: Pranith Karampuri <pranith.karampuri@phonepe.com>
+    P: Ravishankar N <ravishankar@redhat.com>
+    S: Maintained
+    F: xlators/performance/io-threads/
+
+### Leases
+    S: Maintained
```

Comment:
How can it be maintained without maintainers?

Reply:
Valid concern! These are the xlators which are extensively used in fs (ie, every volume uses these), but there are no bugs in these components over the years.

I used similar format used for few existing entries. Just removed the name of some developers who have left the company (ie, their emails are not valid anymore) from these components.

Reply:
Added a **Stable** status